codes/kotlin/chapter_array_and_linkedlist/array.kt

```diff
@@ -0,0 +1,102 @@
+/**
+ * File: array.kt
+ * Created Time: 2024-1-25
+ * Author: dcurtis
+ */
+package chapter_array_and_linkedlist
+
+import java.util.concurrent.ThreadLocalRandom
+
+class array() {
```

Comment:
Please remove the class if it is not necessary.

In Java, every method should be enclosed in a class. But most programming languages do not require it.